crates/next-core/js/src/overlay/internal/container/Errors.tsx

```diff
@@ -136,10 +142,10 @@ export function Errors({ errors }: ErrorsProps) {
   // Reset component state when there are no errors to be displayed.
   // This should never happen, but let's handle it.
   React.useEffect(() => {
-    if (errors.length < 1) {
+    if (errors.length < 1 && issues.length < 1) {
       setDisplayState("hidden");
     }
-  }, [errors.length]);
+  }, [errors.length, issues.length]);
```

Comment:
Setting state in use effect is generally an anti-pattern.

This could be the following instead:
```tsx
let [displayState, setDisplayState] = React.useState<
  "minimized" | "fullscreen" | "hidden"
>("fullscreen");

if (errors.length === 0 && issues.length === 0) {
  displayState = "hidden";
}
```

Reply:
This is kinda already handled at the bottom, so I'll just remove this here